docs/move-overview.md

```diff
@@ -64,59 +64,31 @@ Now let us see how a programmer can interact with these modules and resources in
-    // Move the `coin` resource into the account of `payee`. If there is no
-    // account at the address `payee`, this step will fail
-    // Note that we could have also written `deposit<LBR>` here, but the
-    // compiler is able to infer this for us based on the type of `coin`
-    LibraAccount::deposit(payer, payee, coin);
-  }
+// Use the LibraAccount module published on the blockchain at account address
+// 0x0...1. 0x1 is shorthand that the language pads out to
+// 16 bytes by adding leading zeroes.
+use 0x1::LibraAccount;
+// Use the LBR resource from the LBR module. Naming the resource here makes
+// it possible to reference the resource without the module name.
+use 0x1::LBR::LBR;
+
+fun peer_to_peer_lbr_payment(payer: &signer, payee: address, amount: u64) {
+  // Acquire the capability to withdraw from the payer's account.
+  let payer_withdrawal_cap = LibraAccount::extract_withdraw_capability(payer);
+  // Withdraw a value of `amount` LBR from the payer's account and deposit
+  // it into the account at the address `payee`. This will fail if the
+  // payer's balance is less than `amount` or if there is no account at the
+  // address `payee`. The `metadata` and `metadata_signature` arguments are
+  // not specified, so this could also fail if the transaction requires
+  // that information.
+  LibraAccount::pay_from<LBR>(&payer_withdrawal_cap, payee, amount, x"", x"");
+  // Restore the capability back to the payer's account.
+  LibraAccount::restore_withdraw_capability(payer_withdrawal_cap);
 }
-```
-
-Let us look at a more complex example. In this example, we will use a transaction script to pay multiple recipients instead of just one.
-
-```move
-// Multiple payee example. This is written in a slightly verbose way to
-// emphasize the ability to split a `Libra<LBR>` resource. The more concise
-// way would be to use multiple calls to `LibraAccount::withdraw_from`.
-
-script {
-  use 0x1::Libra;
-  use 0x1::LibraAccount;
-  use 0x1::LBR::LBR;
-  fun main(payer: &signer, payee1: address, amount1: u64, payee2: address, amount2: u64) {
-    let payer_withdrawal_cap = LibraAccount::extract_withdraw_capability(payer);
-    let total = amount1 + amount2;
-    let coin1 = LibraAccount::withdraw_from<LBR>(&payer_withdrawal_cap, total);
-    LibraAccount::restore_withdraw_capability(payer_withdrawal_cap);
-
-    // This mutates `coin1`, which now has value `amount1`.
-    // `coin2` has value `amount2`.
-    let coin2 = Libra::withdraw(&mut coin1, amount2);
-
-    // Perform the payments
-    LibraAccount::deposit(payer, payee1, coin1);
-    LibraAccount::deposit(payer, payee2, coin2)
-  }
 }
 ```
 
-This concludes our "tour" of transaction scripts. For more examples, including the transaction scripts supported in the initial testnet, refer to `libra/language/stdlib/transaction_scripts`.
+For more examples, including the transaction scripts supported in the initial testnet, refer to `libra/language/stdlib/transaction_scripts`.
```

Comment:
Probably want an actual link to this at some point. But not stopping this PR because of that.